targets/ChibiOS/_nf-overlay/os/hal/src/stm32_rng/hal_stm32_rng.c

```
@@ -77,22 +77,13 @@ void rngReleaseModule()
 
 #endif /* RNG_USE_MUTUAL_EXCLUSION == TRUE */
 
-uint32_t rngGenerateRandomNumber()
+bool rngGenerate(size_t size, uint8_t *out)
```

Cppcheck (2.21.0)

[style] The function 'rngGenerate' is never used.

(unusedFunction)